internal/querycoordv2/meta/load_status_cache.go

```diff
@@ -0,0 +1,97 @@
+// Licensed to the LF AI & Data foundation under one
+// or more contributor license agreements. See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership. The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License. You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package meta
+
+import (
+	"sync"
+	"time"
+
+	"github.com/golang/protobuf/proto"
+	"go.uber.org/zap"
+
+	"github.com/milvus-io/milvus-proto/go-api/commonpb"
+	"github.com/milvus-io/milvus/internal/log"
+	. "github.com/milvus-io/milvus/internal/util/typeutil"
+)
+
+const LoadStatusExpireTime = 24 * time.Hour
+
+var GlobalLoadCache *LoadStatusCache
+
+type LoadStatus struct {
+	s *commonpb.Status
+	t time.Time
+}
+
+type LoadStatusCache struct {
```

Comment:
Name the methods to `Get`, `Put`, `Remove` and `TryExpire` make more readability for me